docs-src/content/using-cubic-sdk/connecting.md

```diff
@@ -191,6 +211,23 @@ Please see the Java section of https://grpc.io/docs/guides/auth/ for more detail
 
 {{% /tab %}}
 
+{{% tab "Swift/iOS" %}}
+``` swift
+import Cubic
+import NIOSSL
+
+class CubicConnection {
+    
+    let serverAddress = "demo-cubic.cobaltspeech.com"
+    let serverPort = 2727
+
+    let client = Client(host: serverAddress, port: serverPort, tlsCertificateFileName: "tet.txt", tlsCertificateFormat: .pem)
```

Comment:
is there a typo in tet.txt? What does it stand for?

Reply:
Oops.